### PR TITLE
Roxanne HW01

### PR DESCRIPTION
**Pull request by roxannefelig**
@ncaliskan @iannel

### CODE REVIEW

File: HW1_data_explore.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Viewing Data Structure
+
+```{r penguins structure}
+str(HW1data)
+nrow(HW1data)
+ncol(HW1data)
+plot(HW1data$species) 
+mean(HW1data$bill_length_mm, na.rm=TRUE)
+```
```
Comment (iannel) [inline HW1_data_explore.md]: not sure why but I don't think the output of your code is not viewable in the "rich diff" format

File: README.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
-# Homework Template
+# Homework 1
 
-This is template repository for hosting and posting your homework assignments for Programming with Data.
```
Comment (iannel) [inline README.md:3]: does this mean the syntax was deleted? It's still showing up in the output

**Review by iannel — COMMENTED** [review]
great job!

File: HW1_data_explore.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
```
Comment (ncaliskan) [inline HW1_data_explore.md]: Hi, great job with you work!! I don't think your files are actually in the hw1 folder though you may want to fix that. Other than that I think you did great !!!!!

Comment (roxannefelig) [reply HW1_data_explore.md]: Ugh yeah I was having a lot of trouble with that yesterday, but think I managed to fix it just now? Thanks!!

**Pull request merged**